test/cmd/templates.sh

```diff
@@ -48,8 +45,10 @@ os::cmd::expect_success_and_text 'oc process ADMIN_USERNAME=myuser ADMIN_PASSWOR
 os::cmd::expect_success_and_text 'oc process -f test/templates/testdata/guestbook.json ADMIN_USERNAME=myuser ADMIN_PASSWORD=mypassword'       '"mypassword"'
 # Argument values with commas are honored
 os::cmd::expect_success 'oc create -f examples/sample-app/application-template-stibuild.json'
-os::cmd::expect_success_and_text 'oc process ruby-helloworld-sample MYSQL_USER=myself MYSQL_PASSWORD=my,1%pass'  '"myself"'
-os::cmd::expect_success_and_text 'oc process MYSQL_USER=myself MYSQL_PASSWORD=my,1%pass ruby-helloworld-sample'  '"my,1%pass"'
+os::cmd::expect_success_and_text 'oc process ruby-helloworld-sample MYSQL_USER=myself MYSQL_PASSWORD=my,1%pa=s'        '"myself"'
+os::cmd::expect_success_and_text 'oc process MYSQL_USER=myself MYSQL_PASSWORD=my,1%pa=s ruby-helloworld-sample'        '"my,1%pa=s"'
```

Comment:
looks like this covers at least one of the test suggestions i made above.

I wonder if cmd/templates.sh and cmd/process.sh should be combined into a single test script so we can do some de-duping.

Not for this PR though.